pkg/controller/servicemesh/webhook/controller.go

```diff
@@ -0,0 +1,205 @@
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/maistra/istio-operator/pkg/controller/common"
+	"k8s.io/api/admissionregistration/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+var (
+	log         = logf.Log.WithName("controller_webhook")
+	secretNames = []string{
```

Comment:
insert empty line here, since log and secretNames aren't related? (will also remove the bunch of spaces in the first of the two lines)